arm/Microsoft.DomainServices/AzureADDomainServices/.parameters/aadds.parameters.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
General comment:
Please also add a GitHub & AzureDevOps module pipeline in the corresponding folders:

- Azure DevOps: `.azuredevops/modulePipelines/`
- GitHub: `.github/workflows/`

You can copy-paste any of the existing module pipelines and just update all module specific properties (i.e. primarily the `env` variables and the deployment test parameters